src/Native/CMakeLists.txt

```diff
@@ -179,4 +179,5 @@ function(install_library_and_symbols targetName)
 endfunction()
 
 add_subdirectory(CpuMathNative)
-add_subdirectory(FastTreeNative)
\ No newline at end of file
+add_subdirectory(FastTreeNative)
+add_subdirectory(LDANative)
```

Comment:
>(LDA [](start = 16, length = 4)

Can we change the name of the directory from `LDANative` to `LdaNative`, to be consistent with other usage elsewhere (the build artifact being `LdaNative.dll`, and so on)?